_resources/port1.0/group/python-1.0.tcl

```diff
@@ -577,5 +583,8 @@ pre-test {
         if {$libdirs ne ""} {
             test.env-append PYTHONPATH=[join $libdirs :]
         }
+        if {${python.pep517} && [option python.pep517_backend] eq "maturin"} {
+            test.env-append PYTHONPATH=${destroot}${frameworks_dir}/Python.framework/Versions/${python.branch}/lib/python${python.branch}/site-packages
+        }
```

Comment:
@reneeotten I feel that I don't understand that do you mean. May I ask your help?

Reply:
sorry, I am mistaken about that.... this comes back to the fact that `maturin` as a build-system only generates a `.whl` file and doesn't create a `build/` directory. We had an initial discussion about this when I [added](https://github.com/macports/macports-ports/pull/15001) this stuff to the PG.

From reading the MacPorts guide I am not sure whether your earlier comment that the `test` phase is run after the `destroot` phase is correct. Also when testing some Python ports locally I don't see a `destroot` phase when running `sudo port -dv test pyXY-portname`.

Perhaps @jmroot can provide some insight and/or give some advice here?

Reply:
@reneeotten yeah, I've tried quite long to find something that works. Strange, because I see behaviour like:
```
--->  Computing dependencies for py310-mpd2
--->  Fetching distfiles for py310-mpd2
--->  Attempting to fetch python-mpd2-3.1.0.tar.gz from https://fra.de.distfiles.macports.org/py-mpd2
--->  Attempting to fetch python-mpd2-3.1.0.tar.gz from https://files.pythonhosted.org/packages/source/p/python-mpd2
--->  Verifying checksums for py310-mpd2
--->  Extracting py310-mpd2
--->  Configuring py310-mpd2
--->  Building py310-mpd2
--->  Staging py310-mpd2 into destroot                   
--->  Testing py310-mpd2
```
at each picked port.

Which port have you used to test?

Reply:
The test phase is not declared to require destroot in the current release. I assume you're running master.

Reply:
> The test phase is not declared to require destroot in the current release. I assume you're running master.

yes, I'm. 

@reneeotten when I have no idea how to fix tests for that port. Do you have any idea?